internal/scan-manager/scanners/veracode/src/main/java/org/wso2/security/tools/scanmanager/scanners/veracode/handler/ScanTask.java

```diff
@@ -131,9 +129,7 @@ private boolean handleUploadingTask() {
         }
         if (!isUploadSuccess) {
             logMessage = logMessage.concat(" Terminating scan for app: " + scanContext.getAppId());
-            log.error(logMessage);
-            CallbackUtil.persistScanLog(scanContext.getJobId(), logMessage, LogType.ERROR);
-            CallbackUtil.updateScanStatus(scanContext.getJobId(), ScanStatus.ERROR, null, null);
+            log.error(new CallbackLog(scanContext.getJobId(), logMessage));
```

Comment:
A `scanner` will have only one `ScanContext` for it's live time. If we should have a way of making `ScanContext` available to all everyone we will not have to pass JobId in every log entry. In such case, we will not need the `CallbackLog`class tool. 

If this is a major change, let's create a issue and do this as a separate improvement. WDYT?

Reply:
Created to track this https://github.com/wso2/security-tools/issues/128